service/Service/ServiceConfiguration.cs

```diff
@@ -206,20 +210,20 @@ private void ConfigureIngestionEmbeddingGenerators(IKernelMemoryBuilder builder)
             {
                 case string x when x.Equals("AzureOpenAI", StringComparison.OrdinalIgnoreCase):
                 case string y when y.Equals("AzureOpenAIEmbedding", StringComparison.OrdinalIgnoreCase):
-                {
-                    var instance = this.GetServiceInstance<ITextEmbeddingGenerator>(builder,
-                        s => s.AddAzureOpenAIEmbeddingGeneration(this.GetServiceConfig<AzureOpenAIConfig>("AzureOpenAIEmbedding")));
-                    builder.AddIngestionEmbeddingGenerator(instance);
-                    break;
-                }
+                    {
```

Comment:
@dluc Do you use some kind of formatter on this project? This spaces were added by my vscode and I don't know how to configure it not to do so. Is there any tool in C# world like Prettier is in javascript?

Reply:
Which IDE/editor do you use?

Usually I work with Rider, that integrates Resharper (a VisualStudio extension). Rider and Resharper read and apply settings from two files: `KernelMemory.sln.DotSettings` and `.editorconfig`. 

If you use VS Code with **EditorConfig** you should get most of the formatting, though some might be missing because they come from the `.DotSettings` file.